hr_attendance_autoclose/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+#. Go to *Settings > Company > Attendances*.
```

Comment:
This is no longer true, now we have:
![image](https://github.com/OCA/hr-attendance/assets/3512779/9bf4f460-02b0-4fbd-a00f-75802355565a)

Could you please modify it?